apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/create-page/style.scss

```diff
@@ -13,17 +13,17 @@ $create-page-text-color: #a2aab2;
 	min-height: 45px;
 
 	&:focus {
-		color: $white;
```

Comment:
Just a quick note that `$white` probably didn't break the rule, just because it's referencing a sass variable rather than a named color. Either way, this is a fair change :)